package.json

```diff
@@ -24,17 +24,18 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bower-less": "^0.0.1",
+    "bower-less": "0.0.1",
```

Comment:
i did not do that and i dont know how that happened.
I will revert it. ...strange